Automattic/wp-calypso
# List all availableFor plans for eCommerce plan

#### Changes proposed in this Pull Request

Related to eCommerce plan launch. There was a PR that added new constants to `plans/constants.js` but it didn't list `availableFor` plans - which is a list of upgrade paths. This PR adds that missing bit.

#### Testing instructions

* Pick an eCommerce plan, go to checkout, confirm you can see a subscription length picker:

<img width="732" alt="zrzut ekranu 2018-11-20 o 17 10 53" src="https://user-images.githubusercontent.com/205419/48786479-40966a00-ece7-11e8-8586-fcf36269b510.png">
